pom.xml

```diff
@@ -218,6 +219,10 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>symbol-annotation</artifactId>

```

Comment:
Isn't this part of the structs plugin as well? I already have a dependency on the structs plugin that should provide this.

Reply:
You are absolutely right, I forgot the name of the global lib and only "look" for this one. I'll remove it.